charts/memgraph/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: memgraph
 home: https://memgraph.com/
 type: application
 version: 0.1.7
```

Comment:
The version of the Memgraph chart needs to be also updated, so it should be `0.1.8`, since it will be a new version.